kolibri/utils/options.py

```diff
@@ -385,6 +405,17 @@ def csp_source_list(value):
 
 
 base_option_spec = {
+    "FileStorage": {
+        "STORAGE_BACKEND": {
+            "type": "storage_option",
+            "options": ("file_system", "gcs"),
+            "default": "file_system",
+            "description": """
+            The storage backend class that Django will use when managing files. The class given here must implement
+            the Django files.storage.Storage class.
+            """,
```

Comment:
Let's add addiitonal options here for the ACL, bucket name, etc - we don't need to expose all the settings that are available, but for anything we need set, we can include here.